include/grpcpp/generic/generic_stub_impl.h

```diff
@@ -82,6 +82,12 @@ class GenericStub final {
                    const grpc::ByteBuffer* request, grpc::ByteBuffer* response,
                    std::function<void(grpc::Status)> on_completion);
 
+    /// Setup and start a unary call to a named method \a method using
```

Comment:
delete duplicate "method"

Reply:
The first method is the English word "method" and the second is the parameter `\a method` which gets interpreted by doc builders that we apply at release time.

Reply:
So it's not really a duplicate, is that cool?